src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJobOptions.java

```diff
@@ -125,6 +125,13 @@ public class RevAiJobOptions {
   @SerializedName("segments_to_transcribe")
   private List<SegmentToTranscribe> segmentsToTranscribe;
 
+  /**
+   * Optional and only available with transcriber "human".
```

Comment:
Can you defer this explanation to the documentation site? I don't want to replicate code that we will need to update if we decide to add this to Ai transcription as well.

Reply:
I guess I didn't consider this because I was just following the established pattern. Are you suggesting to delete the line saying that it's only for 'human' entirely and adding a link to the option in our documentation. I'm hesitant to add links because of the potential for broken links.

Reply:
Yeah, up to you. I was just giving another opinion to consider for our docs.
I was trying to optimize for least number of SDK updates to release/ or change how a feature works

Reply:
I guess now I'm wondering if there should be anything at all. Should we just delete the 'only for human' lines entirely and not replace them with anything? We don't mention similar restrictions on foreign languages or hipaa

Reply:
You could continue with what you have and we can discuss it async at our next office hours

Reply:
I'll bring it up at OH